### ISSUE TITLE
Crossbow and arrow projection issues

### ISSUE THREAD

**Issue by LichLord91** [GeyserMC/Geyser#1939]
**Describe the bug**

Arrows from bows have a slight but noticeable delay when fired 

Crossbows loading never stops but does load if you let go eventually. Also when shooting the area it seems to disappear completely. 

**To Reproduce**

For Bow:
1. Load and shoot bow with arrow

For Crossbow:
1. Try to load crossbow
2. see that it doesn't stop the loading animation
3. let go of loading button
4. shoot arrow

**Expected behavior**

For bows fast and accurate projection

For the crossbow proper load animations and readying.
Arrow to be seen after shooting.

**Screenshots / Videos**

For bow:
https://streamable.com/5lkui0

For crossbow:
https://streamable.com/p4bw33

**Server Version and Plugins**

"version": "1.2.0-SNAPSHOT (git-server-inventory-ce33ca5)", Single instance Geyser-Spigot.jar

**Geyser Dump**

https://dump.geysermc.org/qpUF73Lt0XTDCu9j3hH7mu9AtVrDaVCy

**Minecraft: Bedrock Edition Version**

v1.16.201 on Windows 10 a

May be related to

https://github.com/GeyserMC/Geyser/issues/1223
https://github.com/GeyserMC/Geyser/issues/500

**Comment by Camotoy** [comment]
The latter issue is a result of differing mechanics between Bedrock and Java - Java will always continuously load the crossbow; Bedrock will stop when fully charged. The current codebase matches Java behavior, even if the animation isn't a fan.

**Comment by LichLord91** [comment]
> The latter issue is a result of differing mechanics between Bedrock and Java - Java will always continuously load the crossbow; Bedrock will stop when fully charged. The current codebase matches Java behavior, even if the animation isn't a fan.

Ahh i see but what about the arrows completely disappearing after shooting? 

Also i realized i had a bunch of resource packs installed so i unloaded them to double confirm the issue and it was all still the same so no issue on the resource pack side.

**Comment by LichLord91** [comment]
> > The latter issue is a result of differing mechanics between Bedrock and Java - Java will always continuously load the crossbow; Bedrock will stop when fully charged. The current codebase matches Java behavior, even if the animation isn't a fan.
> 
> Ahh i see but what about the arrows completely disappearing after shooting?
> 
> Also i realized i had a bunch of resource packs installed so i unloaded them to double confirm the issue and it was all still the same so no issue on the resource pack side.

Scratch that completely disappear just did some more testing i see that they shoot just the animation is hard to follow and upon initial shooting the arrow kinda jiggles all over the place

**Comment by Camotoy** [comment]
> upon initial shooting the arrow kinda jiggles all over the place

This should be fixed in the next build.

**Comment by LichLord91** [comment]
Awesome thank Camotoy.

**Comment by LichLord91** [comment]
still awaiting merge into server-inventory branch to test. :)

**Comment by LichLord91** [comment]
> > upon initial shooting the arrow kinda jiggles all over the place
> 
> This should be fixed in the next build.

Hey Camotoy,

Just gave this a whirl in latest push , still experiencing same issue 
1.2.0-SNAPSHOT (git-server-inventory-13c924f)"

https://streamable.com/gpmqac

**Comment by Jens-Co** [comment]
@LichLord91 hello there, Do you still encounter this bug on the latest master build of geyser ? Let me know if you do,thanks!

**Comment by Jens-Co** [comment]
I'll close this issue since there is no response and I'm unable to replicate this issue on latest geyser master. If somehow you are still getting this error on the latest, please let me know and I'll reopen this issue. Thanks.
